frontend/packages/client/src/contexts/NotificationService.js

```diff
@@ -84,54 +66,37 @@ const NotificationServiceProvider = ({ children }) => {
 
   const getUserSettings = async () => {
     try {
-      //here we call api
-      const { communitySubscription, isSubscribedFromCommunityUpdates } =
-        INIT_NOTIFICATION_SETTINGS;
+      const { communitySubscription, isSubscribedFromCommunityUpdates, email } =
+        await getUser(addr);
       setNotificationSettings((prevState) => ({
         ...prevState,
         communitySubscription,
         isSubscribedFromCommunityUpdates,
+        email,
       }));
     } catch {
       throw new Error('cannot get user settings');
```

Comment:
in case `getUserSettings` error, will user see error modal when they land on Settings page?

Reply:
Not right now, if for any reason any of the user init functions failed (startLeanplum, setUserId, getUserSetting) we are only console logging the error. However, on the flow of updating subscriptions, getUserSetting error is caught and error modal shows up

Reply:
i see, if `getUserSettings` failed, how is the current flow look like when user visit Settings page? Is it possible to display the error modal for that case too?